modules/migration/migration-resources/5.8.0/dbscripts/step1/identity/postgresql.sql

```diff
@@ -1 +1,21 @@
 ALTER TABLE IDN_CLAIM_PROPERTY RENAME LOCAL_CLAIM_ID TO CLAIM_ID;
+
+DROP TABLE IF EXISTS IDN_AUTH_USER;
+CREATE TABLE IDN_AUTH_USER (
+	USER_ID VARCHAR(255) NOT NULL,
+	USER_NAME VARCHAR(255) NOT NULL,
+	TENANT_ID INTEGER NOT NULL,
+	DOMAIN_NAME VARCHAR(255) NOT NULL,
+	IDP_ID INTEGER NOT NULL,
+	PRIMARY KEY (USER_ID),
+	CONSTRAINT USER_STORE_CONSTRAINT UNIQUE (USER_NAME, TENANT_ID, DOMAIN_NAME, IDP_ID));
+
+DROP TABLE IF EXISTS IDN_AUTH_USER_SESSION_MAPPING;
```

Comment:
in which version this table was introduced ?

Reply:
For IS 5.2.0 and 5.7.0 we have patched. For IS 5.8.0 it will be added by default. Tables are added at https://github.com/wso2/carbon-identity-framework/pull/2020

Reply:
In that case dropping the tables is not correct. If the user already has this in older version, they'll lose all the data.

The other concern is, we have to check whether the table exist before adding it.

Reply:
fixed